app/tests/test.py

```diff
@@ -4,68 +4,75 @@
 import os
 
 is_travis = 'TRAVIS' in os.environ
+
+
 class BadgeyayTest(unittest.TestCase):
 
-	@classmethod
-	def setUpClass(cls):
-		if is_travis:
-			cls.driver = webdriver.PhantomJS()
-		else:
-			cls.driver = webdriver.Firefox()
-		cls.driver.get('http://localhost:5000')
-		super(BadgeyayTest, cls).setUpClass()
+    @classmethod
+    def setUpClass(cls):
+        if is_travis:
+            cls.driver = webdriver.PhantomJS()
+        else:
+            cls.driver = webdriver.PhantomJS()

```

Comment:
I think this def was made to check for using either firefox or phantomjs? if firefox is discarded,then i think is there  need for setupclass()?

Reply:
@S2606 I am checking, I will revert to firefox 😅